### PR TITLE
fix(social-btn): add white border on dark background

### LINKED ISSUE #1121: Docs > Components > Buttons > Dark background: Fix border color for social buttons on active state
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

On https://boosted.netlify.app/docs/5.1/components/buttons/#dark-background, the value of border-color for social inverse buttons (= with a dark background) active state don't change, so on mouse down, visually, it seems that the buttons get smaller.

### Reduced test cases

NA

### What operating system(s) are you seeing the problem on?

Windows, macOS

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Boosted are you using?

v5.1.0

### PR DESCRIPTION
**Pull request by hannahiss**
fix(social-btn): on dark background,need a white border so that the size of button doesn't decrease.
Closes #1121

### CODE REVIEW

**Review by louismaximepiton — APPROVED** [review]
LGTM 🚀

File: scss/_buttons.scss [hunk 1]
```diff
@@ -274,6 +274,7 @@
       &:active {
```
Comment (julien-deramond) [inline scss/_buttons.scss:274]: While I was testing the PR, I tried to add `.active` on those buttons and their rendering is not correct:
Screenshot from 2022-04-01 12-17-29

We could improve the selector by doing:

```scss
&:active,
&.active {
```

**Review by julien-deramond — CHANGES_REQUESTED** [review]

File: scss/_buttons.scss (hunk 1 above)
Comment (hannahiss) [reply scss/_buttons.scss]: That was not in the PR ;-) but ok :-)

**Review by julien-deramond — APPROVED** [review]

**Comment by julien-deramond** [comment]
@hannahiss Could you backport it in v4 as well?

**Pull request merged**